tests/fedora-str/playbook-run.sh

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/bash
+# A thin wrapper for ansible-playbook which has a nice check for
+# TEST_SUBJECTS being set.
 set -xeuo pipefail
 
 dn=$(cd $(dirname $0) && pwd)
```

Comment:
```
+ exec ansible-playbook --tags=atomic tests/fedora-str/sysinstall-tests.yml
ERROR! the playbook: tests/fedora-str/sysinstall-tests.yml could not be found
### EXITED WITH CODE 1 AFTER 1s
```

Hmm, probably we should just drop the `cd` below this line to make it easier to pass playbooks to this.